src/advanced_inv.cpp

```diff
@@ -1681,6 +1683,24 @@ void advanced_inventory::display()
                 continue;
             }
             action_examine( sitem, spane );
+        } else if( action == "EXAMINE_CONTENTS" ) {
+            if( sitem == nullptr ) {
+                continue;
+            }
+            item_location sitem_location = sitem->items.front();
+            bool contents_to_examine = false;
+            if( !sitem_location->is_container_empty() ) {
```

Comment:
This block of code is identical to the one in `inventory_ui` and could perhaps be moved to a helper. It's not a big deal though

Reply:
I agree that the duplicate code here isn't ideal.  I'm contemplating various ways to simplify the use of various inventory_selector classes (there seems to be a similarly-duplicated block of code for each of them), but I think that's a longer-term goal.